setup.py

```diff
@@ -57,7 +57,7 @@ def run(self):
 
 
 setup(name='citrine',
-      version='0.72.3',
+      version='0.73.1',
```

Comment:
Why the minor/patch version bump?

Reply:
uhhh... good question... that was a merge failure on my part. I had some code for a different ticket hanging around that shouldn't have gotten in, and got confused when it was in my merge conflict. This should be `0.72.4`

Reply:
Fixed, ended up with a minor patch since this introduces a backwards compatible new feature